### DISCUSSION TITLE
React native error in WebRTC Play

### DISCUSSION THREAD

**Discussion by vincemavill** [ant-media/Ant-Media-Server#6166] [Q&A]
React native error in WebRTC Play where says "Cannot read property 'toURL' of undefined"
line 55 from https://github.com/ant-media/WebRTC-React-Native-SDK/blob/main/example/src/Play.tsx

i am using WebRTC Multitrack Conference for publishing stream.
"@antmedia/react-native-ant-media": "^1.0.6",
"react-native": "0.73.4",
  
JDK: 17
NODEJS: 18.17.1

![image (8)](https://github.com/ant-media/Ant-Media-Server/assets/18587242/d3b3a7a0-9f26-4b14-8a68-98f4949f6d59)

**Comment by USAMAWIZARD** [comment]
@vincemavill , I have just tried the play example and its working fine for me , can you please let me know what you are trying to archive ? play example is there for playing a single stream not a conference stream which has multiple tracks , please use conference example and join as a second user if you want to see the stream of other participants in a conference call.

**Reply by vincemavill** [reply]
@USAMAWIZARD i am trying also the publisher in the web app browser then the play example in react native, but it shows the error like that i can only hear the audio not the video.

**Reply by vincemavill** [reply]
Please dont download the whole git project, just copy the examples component and insert it in the existing project. so you can replicate the issues i encountered @USAMAWIZARD

**Reply by USAMAWIZARD** [reply]
version 1.0.6 does not  support multi track conference , it was based on old conference structure , I would recommend you use it from github , let me sync with my colleague when is the next version expected to be released. which will support multi track conference.

**Reply by vincemavill** [reply]
@USAMAWIZARD that make sense now, thank you. that is why we are using the 1.0.5 and its working fine.
